data/prohibited-countries.json

```diff
@@ -0,0 +1,31 @@
+{
+  "United States": "US",
+  "Cuba": "CU",
+  "North Korea": "KP",
+  "Syria": "SY",
+  "Iran": "IR",
+  "Russia": "RU",
+  "Ukraine": "UA",
+  "Albania": "AL",
```

Comment:
@DavidMinarsch being conservative, we're still blocking countries from the second part of the list [here](https://orpa.princeton.edu/export-controls/sanctioned-countries). If you're certain we're OK to drop it to just the 'comprehensively sanctioned' countries, we can simply remove them here.

Reply:
Upon further review it looks too heavy-handed to block the countries in the second list, and this is what you said in the [doc David](https://docs.google.com/document/d/1PPygoAhnJfQcxgZCd8BcoxOx33bpC8nnR_MfohzOW2w/edit?disco=AAAA-lwHZyU). So @mohandast52, please remove all countries from Albania through to Libya. The list should only contain US, Cuba, NK, Syria, Iran, Russia and Ukraine.